docs/ibc/proposals.md

```diff
@@ -0,0 +1,39 @@
+<!--
+order: 5
+-->
+
+# Governance Proposals
+
+In uncommon situations, a highly valued client may become frozen due to uncontrollable 
+circumstances. A highly valued client, might have hundreds of channels being actively used.
+Some of those channels might have a significant amount of locked tokens used for ICS 20.
+
+If the one third of the validator set the chain the client represents decides to collude,
+they can sign off on two valid but conflicting headers each signed by the other one third
+of the honest validator sets. The light client can now be updated with two valid, but conflicting
+headers at the same height. The light client cannot know which header is trustworthy and therefore
+evidence of such misbehaviour is likely to be submitted resulting in a frozen light client. 
+
+Frozen light clients cannot be updated under any circumstance except via a governance proposal.
+Since validators can arbitarily agree to make state transitions that defy the written code, a 
+governance proposal has been added to ease the complexity of unfreezing or updating clients
+which have become "stuck". Unfreezing clients, re-enables all of the channels built upon that
+client. This may result in recovery of otherwise lost funds. 
+
+Tendermint light clients may also become expired if the trusting period has passed since their 
+last update. If a chain undergoes an unplanned upgrade, there may be no commitment to that upgrade
+signed by the validator set before the chain-id changes. In this situation, the validator set of
+the last valid update for the light client is never expected to produce another valid header since 
+the chain-id has changed, which will ultimately lead the on-chain light client to become expired.  
```

Comment:
Can this be two separate paragraphs since they're two separate situations?

Reply:
What is two separate situations? I was intending to show that unplanned upgrades results in client expiry

Reply:
I also don't understand the question

Reply:
Oh, do you mean that they could be expired just because no headers have been submitted even if there was no unplanned upgrade?

Reply:
Yeah, that should be clarified.

Reply:
Updated to separate general expiry from expiry due to unplanned upgrade by the counterparty chain